include/boost/gil/bit_aligned_pixel_reference.hpp

```diff
@@ -35,10 +30,10 @@ namespace boost { namespace gil {
 //  Represents a range of bits that can span multiple consecutive bytes. The range has a size fixed at compile time, but the offset is specified at run time.
 /////////////////////////////
 
-template <int RangeSize, bool Mutable>
+template <int RangeSize, bool IsMutable>
 class bit_range {
 public:
-    using byte_t = typename mpl::if_c<Mutable,unsigned char,const unsigned char>::type;
+    using byte_t = typename mp11::mp_if_c<IsMutable, unsigned char, unsigned char const>;
```

Comment:
A bogus `typename`, is not it?

Reply:
Indeed. Good catch !

Reply:
@Kojoley Thanks for spotting this. I've removed this one and a few more left over.